src/aom/headers.js

```diff
@@ -0,0 +1,155 @@
+// @ts-check
+// Module aom/headers
+// Generate the headers material based on the provided configuration.
+// CONFIGURATION
+//  - specStatus: the short code for the specification's maturity level or type (required)
```

Comment:
If you would like, we can add AOM specific section to the documentation. This is what we do for w3c:

https://respec.org/docs/#w3c-specific-configuration-options